static/refcache.json

```diff
@@ -1547,6 +1551,10 @@
     "StatusCode": 200,
     "LastSeen": "2023-02-20T07:47:26.390977-05:00"
   },
+  "https://github.com/ecosystem/registry/": {
+    "StatusCode": 404,
+    "LastSeen": "2023-05-04T10:07:23.310712+02:00"
+  },
```

Comment:
I think that this entry is an error. Delete it if it is.

```suggestion
```

Reply:
yes